dockerfiles/entrypoints/createbuckets.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Exit if we are not initializing the setup
+test "${INIT}" || exit 0
+
+/wait-for-it.sh storage:9000 --timeout=30
```

Comment:
I think we already had a "wait for it" command...

Found it! It's called `wait_for_search.py` (https://github.com/rtfd/readthedocs-common/blob/baa26aa89229b0a59a9507eb5992e559d37775b1/dockerfiles/scripts/wait_for_search.py#L1). If we are sold on this new `wait-for-it.sh`, we should replace our custom one with this one to keep consistency.

Reply:
I can do that, yes

The idea of `wait-for-it.sh` is to have as few requirements as possible AFAIK. It also exists as a `debian` package so not all images will necessarily need to include the shell script.

Reply:
`wait_for_search.py` is now replaced with `wait-for-it.sh`

`wait_for_search.py` might also have waited indefinitely since there was no `timeout=...` passed to `requests.get()` -- I found that it's necessary to wait for quite a long time for `search` to be ready... ~ 1 minute on my setup.